Sliding-Images-Easy.md

```diff
@@ -0,0 +1,247 @@
+# Sliding Images Easy
+
+This tutorial will walk you through building an image slider using the [jQuery](https://jquery.com/) library.
+
```

Comment:
Add a VERY LARGE image here showing the slides in a browser interface and when clicked, it should go to the demo codepen or jsbin or whatever.